src/Features/CSharp/Portable/ExtractMethod/CSharpSelectionResult.cs

```diff
@@ -95,6 +97,37 @@ protected override bool UnderAsyncAnonymousMethod(SyntaxToken token, SyntaxToken
                    current.GetLastToken().Span.End <= lastToken.Span.End;
         }
 
+        protected sealed override bool IsConfigureAwaitFalse(SyntaxNode node)
+        {
+            if (node is InvocationExpressionSyntax
```

Comment:
i don't mind the huge pattern.  i just would prefer it not be in the if-condition.  can you just extract it and assign to a local?